prow/tide/tide.go

```diff
@@ -457,6 +457,7 @@ func contextsToStrings(contexts []Context) []string {
 	for _, c := range contexts {
 		names = append(names, string(c.Context))
 	}
+	sort.Strings(names)
```

Comment:
Why sorting here, Is that because it helps in UT?

Reply:
The contexts are sorted for the following reasons:
1. Helps in UT. 
2. Improves readability in logs as the contexts are sorted.

Reply:
please add this as comment

Reply:
Added comment as suggested.